preprocessing/tripadvisorPreProc.py

```diff
@@ -0,0 +1,27 @@
+import csv
+f = open('tripadvisor_in-restaurant_sample.csv', encoding="UTF-8")
+output = open('tripadvisor_data_processed_USA.csv', 'w', encoding="UTF-8")
+csv_f = csv.reader(f)
+
+count = 0
+
+for row in csv_f:
+    if (row[7] == 'United States'):
```

Comment:
The processed file is missing a header line that specifies attributes (i.e. `Restaurant ID,Restaurant URL,Name,....`) because of this `if` condition. Can you fix this?